### ISSUE TITLE
'Run Selected Tests' and 'Debug Selected Tests' start in different directory

### ISSUE THREAD

**Issue by candychiu** [etas/vs-boost-unit-test-adapter#225]
Project setting:
Debugging/Working Directory:   $(ProjectDir)

Inside the code, I tried to print out Currrent Directory at test start up:
	wchar_t buffer[MAX_PATH];
	GetCurrentDirectory(MAX_PATH, buffer);
	wcout << buffer << endl;

I noticed that if the test was launched in the 'Test Explorer' by 'Run Selected Tests', Current Directory prints the value of $(ProjectDir).   If test was launched by 'Debug Selected Tests', Current Directory prints the value of $(OutDir).  

Project was compiled under x64.  Test -> Test Settings -> Default Processor Architecture has been set to x64.

**Comment by netspiri** [comment]
Hi candychiu and thanks for the feedback.

I tested the following with the latest Visual Studio 2015 release of the add-on available from the gallery and could not reproduce the issue.

```c++
#define BOOST_TEST_MODULE SAMPLE_PROJECT
#include <boost/test/unit_test.hpp>

#define WIN32_LEAN_AND_MEAN
#include <Windows.h>

#include <iostream>

BOOST_AUTO_TEST_CASE(GetWorkingDirectory)
{
	wchar_t buffer[MAX_PATH];
	GetCurrentDirectory(MAX_PATH, buffer);
	std::wcout << buffer << std::endl;
}
```

These are my project properties based on configuration:
- Release/Win32 - Debugging/Working Directory: `$(TargetDir)`
- Debug/Win32 - Debugging/Working Directory: `$(ProjectDir)`

When I switched the configuration (see below), the behaviour was as expected - running/debugging the test via the right-click context menu worked as expected and the correct working directory was noted.

![solutionconfiguration](https://user-images.githubusercontent.com/6962427/33539478-ef5fd232-d8c6-11e7-8c45-7a5aa7655905.png)

Please note that this may not always work due to connection issues. The logs should note this when failing to properly connect to the Visual Studio running instance.

In conclusion, as noted in the [readme.md](https://github.com/etas/vs-boost-unit-test-adapter/blob/master/README.md), support for the 2015 version of the adapter has been dropped in favor of the 2017 version hosted by [Microsoft](https://github.com/microsoft/vs-boost-unit-test-adapter). We recommend that such issues are noted in their fork should this issue be reproducible on their version of the adapter as well. Should this issue persist and you would still like to use the 2015 version, please let us know.

**Comment by candychiu** [comment]
@netspiri 
I encountered this issue in x64 only.  Didn't see it in Win32.  Would you try x64 to see whether you can reproduce it?
